.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Please avoid committing `.vscode` files to the repo. You can edit a global `.gitignore` file and configure `.git` to use that; alternatively, you can edit `.git/info/exclude` inside this repo to ignore the `.vscode` folder.